src/database/utils.tsx

```diff
@@ -4,9 +4,11 @@ import { EpidemicModelPersistent } from "../impact-dashboard/EpidemicModelContex
 
 const undefinedString = "undefined";
 
-export const prepareForStorage = (state: EpidemicModelPersistent): object => {
+export const prepareForStorage = (
+  stateName: EpidemicModelPersistent,
+): object => {
```

Comment:
the ones in this file should remain as `state` I think ... they refer to application state rather than the  name of a US state